src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -8,16 +8,11 @@
             {{ "form_not_available"|trans({}, "SonataAdminBundle") }}
         </div>
     {% else %}
-        <form
-              {% if sonata_config.getOption('form_type') == 'horizontal' %}class="form-horizontal"{% endif %}
```

Comment:
This is missing

Reply:
This is some kind of bugfix, because we now respect the form theme of the admin